src/test/runTest.ts

```diff
@@ -16,7 +16,8 @@ async function main() {
 		const extensionTestsPath = path.resolve(__dirname, './start');
 
 		// Download VS Code, unzip it and run the integration test
-		await runTests({ extensionDevelopmentPath, extensionTestsPath });
+		// NOTE: Using 1.75.0 explicitly as newer versions seem to be crashing on startup
+		await runTests({ version: "1.75.0", extensionDevelopmentPath, extensionTestsPath });

```

Comment:
This seems unfortunate; that's 3 versions old. Is there an issue to cross link here? Something we can tell the vscode team?

Reply:
I haven't had time to really dig into it, but I think it may be a crash due to Electron/Node ABI changes. For now I just want to get end to end testing working.